src/Form.scss

```diff
@@ -1,18 +1,29 @@
 form {
   display: grid;
   grid-template-columns: max-content max-content;
-  grid-gap: 0.5em;
+  grid-gap: 0.75em;
+  margin-bottom: 0.75em;
+
+  &.wide-first-column {
+    grid-template-columns: 475px max-content;
+
+    fieldset {
```

Comment:
I feel like since this is wide-first-column, this should be fieldset:first-child and then spec-fields doesn’t need to specify width